src/intl/en/page-wallets.json

```diff
@@ -1,20 +1,20 @@
 {
-  "page-wallets-accounts-addresses": "Wallets, accounts, and addresses",
+  "page-wallets-accounts-addresses": "Wallets, accounts, keys and addresses",
   "page-wallets-accounts-addresses-desc": "It's worth understanding the differences between some key terms.",
-  "page-wallets-accounts-ethereum-addresses": "An Ethereum account has an Ethereum address, like an inbox has an email address. You can use this to send funds to an account.",
+  "page-wallets-accounts-ethereum-addresses": "An Ethereum account has an address, like an inbox has an email address. This is used to identify your digital assets.",
   "page-wallets-alt": "Illustration of a robot with a vault for a body, representing an Ethereum wallet",
-  "page-wallets-ethereum-account": "An Ethereum account is an entity that can send transactions and has a balance.",
+  "page-wallets-ethereum-account": "An Ethereum account is really a pair of keys - one becomes an address that you can share freely and the other must be kept secret and can be used to make signatures. Together, these keys allow you to hold assets and make transactions.",
   "page-wallets-blog": "Coinbase blog",
   "page-wallets-bookmarking": "Bookmark your wallet",
   "page-wallets-bookmarking-desc": "If you use a web wallet, bookmark the site to protect yourself against phishing scams.",
   "page-wallets-cd": "Physical hardware wallets are devices that let you keep your crypto offline – very secure",
-  "page-wallets-desc-2": "You need a wallet to send funds and manage your ETH.",
+  "page-wallets-desc-2": "Wallets are what most people use to handle their digital assets and identity.",
   "page-wallets-desc-2-link": "More on ETH",
-  "page-wallets-desc-3": "Your wallet is only a tool for managing your Ethereum account. That means you can swap wallet providers at any time. Many wallets also let you manage several Ethereum accounts from one application.",
-  "page-wallets-desc-4": "That's because wallets don't have custody of your funds, you do. They're just a tool for managing what's really yours.",
-  "page-wallets-description": "Ethereum wallets are applications that let you interact with your Ethereum account. Think of it like an internet banking app – without the bank. Your wallet lets you read your balance, send transactions and connect to applications.",
+  "page-wallets-desc-3": "Your wallet is a tool for interacting with your Ethereum account. That means you can swap wallet providers at any time. Many wallets also let you manage several Ethereum accounts from one application.",
+  "page-wallets-desc-4": "Wallet providers don't have custody of your funds, you do, because you have safely stored your keys. The wallet is just a software that helps you control your keys and manage your assets.",
```

Comment:
```suggestion
  "page-wallets-desc-4": "Wallet providers don't have custody of your funds, you do, because you have safely stored your keys. The wallet is just software that helps you control your keys and manage your assets.",
```

Reply:
First sentence also a bit clunky

Reply:
```suggestion
  "page-wallets-desc-4": "Wallet providers don't have custody of your funds. They just provide you a window to see your assets on Ethereum and tools to easily manage them.",
```